### PR TITLE
Translation support

### PR DESCRIPTION
**Pull request by cfoellmann**
These are some changes to employ best practices and fixes to allow for proper translation support and fix support for translate.wordpress.org

### CODE REVIEW

**Comment by mikeselander** [comment]
@cfoellmann thanks for this and all of the other issues/merge requests! I'll try to take a look at these this week or early next week and get some stuff merged 👍

**Comment by cfoellmann** [comment]
Thanks for giving me a heads up.
I will post some more issues and code.

Could you please give my ideas a thumbs up or down so I know if the idea is good for you or something you don’t want to integrate

File: admin/manager-page.php [hunk 1]
```diff
@@ -272,8 +272,8 @@ public function add_menu_item() {
 
 		$page = add_submenu_page(
 			'tools.php',
-			__( 'Manage WP Options Table', 'options_editor' ),
-			__( 'Manage wp_options', 'options_editor' ),
+			__( 'Manage WP Options Table', 'wp-options-editor' ),
```
Comment (mikeselander) [inline admin/manager-page.php:275]: Why the change to `wp-options-editor`?

**Comment by cfoellmann** [comment]
Best practice:

https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains

I am a little bit ocd when it comes to code. But it is just to prevent text-domain interferenxe

**Review by mikeselander — APPROVED** [review]
Looks good 👍
